src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -250,5 +252,71 @@ private static IEnumerable<ProjectContext> GetMatchingProjectContexts(IEnumerabl
                 }
             }
         }
+
+        private static void CopyContents(ProjectContext context, string outputPath)
+        {
+            var sourceFiles = context.ProjectFile.Files.GetContentFiles();
```

Comment:
When you call this sourceFiles it gives me the impression of actual source files, like cs files. Maybe we should call it contentFiles?